mandatory/2-function-creation.js

```diff
@@ -15,26 +20,52 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice > 200){
+    return totalPrice = totalPrice - ((totalPrice * 10)/100);
+  } else if(totalPrice < 200){
+    return totalPrice = totalPrice - ((totalPrice * 5)/100);
+  }
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) {
+  let num = 1;
+  while (num <= limit){
+    console.log(num)
+    num = num +2;
+  }
+}
 
 /*
   Complete the buyTwoGetTheCheapestFree function: if user buys two items, the cheapest item will be free!
   The function should return the price to be paid once the discount is applied.
 */
```

Comment:
When you're writing a function - it's a good idea to think about what the different values of the parameters can be.
In this case, what happens if price1 is the same as price2?